io/net/virt-net/lpm.py

```diff
@@ -47,6 +47,28 @@ def setUp(self):
         set up required packages and gather necessary test inputs
         '''
         self.install_packages()
+        self.using_peer = False
+        if self.params.get("lpar", default='') and \
+                            self.params.get("lpar_ip", default=''):
+            self.lpar = self.params.get("lpar", default='')
+            self.lpar_ip = self.params.get("lpar_ip", default='')
+            self.lpar_user = self.params.get("lpar_user", default='root')
+            self.lpar_pwd = self.params.get("lpar_pwd", "*", 
+                                            default='********')
+            self.peer_session = Session(self.lpar_ip, user = self.lpar_user, 
+                                        password = self.lpar_pwd)
+            if not self.peer_session.connect():
+                self.cancel("failed connecting to peer lpar")
+            self.using_peer = True
+        else:
+            self.lpar = self.get_partition_name("Partition Name")
+            self.log.info("'lpar' parameter empty, migrating current LPAR")
+            if not self.lpar:
+                self.cancel("LPAR Name not got from lparstat command")
+            self.lpar_ip = self.get_mcp_component("MNName")
+            self.log.info("'lpar_ip' parameter empty, migrating current LPAR")
```

Comment:
Get rid of both of these info lines, if I asked you to add that earlier my bad, it's too much info looking at it